pong/gameServer.h

```diff
@@ -2,24 +2,38 @@
 #define GAME_SERVER
 
 #include <pthread.h>
+#include "hardware/lcd.h"
+#include "gameWrite.h"
+#include "playerClient.h"
+
 #define BOARD_SIZE 16
 #define PADDLE_SIZE 3
 
 /**
  * Game server would read the playerClient desired move ie up/down/nothing and update game state
  * and update the display
 */
+typedef struct OutputHardware {
+    LcdHardware ledScreen;
+    MatrixHardware matrix;
+} OutputHardware;
 
-typedef struct gameServer {
+typedef struct GameServer {
     pthread_t pId;
     int board[BOARD_SIZE][BOARD_SIZE];
     int scoreLeft;
     int scoreRight;
     int directionX;
```

Comment:
can we rename these direction vars to smth that indicates that it's the ball direction